content/javascript/concepts/npm/npm.md

```diff
@@ -53,3 +53,25 @@ Alternatively, run the following to display a (long) list of all NPM commands pa
 ```bash
 npm -l
 ```
+
+## Global Packages
+
+NPM also features the ability to install packages globally. This allows you to use packages as command line tools anywhere on your computer.
+
+To install a package globally, use the `-g` flag:
```

Comment:
```suggestion
The `-g` flag is used to install a package globally:
```

Another thing we aim for in Docs is a declarative text that simply states what some is or does or results from, etc.